### PR TITLE
added a more precise description in README files

### PR DESCRIPTION
**Pull request by sdk-banuba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,21 +15,41 @@ Examples for [Banuba SDK on Android](https://docs.banuba.com/face-ar-sdk-v1/andr
...
-   >`low_level_api/include` => `libs/agora_rtc_sdk/include`
-10. Copy and Paste your Banuba client token into appropriate section of `app/src/main/java/com/banuba/sdk/agorapluginexample/ClientToken.kt` with " " symbols. For example: BANUBA_CLIENT_TOKEN = "place_your_token_here"
-11. Visit [agora.io](https://console.agora.io) to sign up and get token, app ID and channel ID.
-12. Copy and Paste your Agora token, app ID and channel ID into appropriate section of `app/src/main/java/com/banuba/sdk/agorapluginexample/ClientToken.kt` with " " symbols. For example: AGORA_APP_ID = "place_your_token_here"
-13. Open the project in Android Studio and run the necessary target using the usual steps.
+   >`banuba_effect_player-release.aar` => [`libs/`](./libs)
+4. Copy `include/bnb` directory into [`libs/bnb_sdk/`](./libs/bnb_sdk):
+   >`include/bnb` => [`libs/bnb_sdk/`](./libs/bnb_sdk)
+5. Download the effects you want from [HERE](https://docs.banuba.com/face-ar-sdk-v1/overview/demo_face_filters). Unzip the downloaded `.zip` files with effects to a folder [`app/src/main/assets/effects`](./app/src/main/assets/effects). Each effect is one folder.
```
Comment (dmitrystoyanovsky) [inline README.md:21]: Download the effects you need from ....
"Each effect is one folder." ->
Each unpacked effect should be put into a separate folder. The folder name will be the effect name for loading.

File: README.md [hunk 2]
```diff
@@ -15,21 +15,41 @@ Examples for [Banuba SDK on Android](https://docs.banuba.com/face-ar-sdk-v1/andr
...
+6. Get the Agora Video SDK archive for android (v4.0.0-beta.2) from [HERE](https://docs.agora.io/en/extension_customer/downloads?platform=Android).
+7. Copy `.jar` files from the Agora Video SDK archive into [`libs`](./libs) dir:
+   >`agora-rtc-sdk.jar` => [`libs/`](./libs)
+8. Copy architecture folders from the Agora Video SDK archive into [`jniLibs`](./libs/jniLibs) dir:
+   >`arm64-v8a`, `armeabi-v7a` and `x86_64` => [`libs/jniLibs/`](./libs/jniLibs)
+9. Copy the `low_level_api/include` directory from Agora SDK into [`libs/agora_rtc_sdk/`](./libs/agora_rtc_sdk/):
+   >`low_level_api/include` => [`libs/agora_rtc_sdk/`](./libs/agora_rtc_sdk/)
+10. As a result of the previous points, the [`libs`](./libs) folder will contain:
```
Comment (dmitrystoyanovsky) [inline README.md:29]: previous steps

File: README.md [hunk 3]
```diff
@@ -15,21 +15,41 @@ Examples for [Banuba SDK on Android](https://docs.banuba.com/face-ar-sdk-v1/andr
...
+libs/agora-rtc-sdk.jar                  - the 'agora-rtc-sdk.jar' file can be found in the archive: (Agora RTC archive)/rtc/sdk/
+libs/banuba_effect_player-release.aar   - the 'banuba_effect_player-release.aar' file can be found in the archive: (Banuba SDK archive)/BNBEffectPlayer/
+```
+11. Copy and Paste your Banuba client token into appropriate section of [`ClientToken.kt`](./app/src/main/java/com/banuba/sdk/agorapluginexample/ClientToken.kt) with " " symbols.
+12. Visit [agora.io](https://console.agora.io) to sign up and get [Agora token, Agora app ID and Agora channel ID](https://docs.agora.io/en/Agora%20Platform/channel_key?platform=Android). You can read more about how you can get an Agora token at [THIS LINK](https://docs.agora.io/en/Agora%20Platform/channel_key?platform=Android).
+13. Copy and Paste your Agora token, Agora app ID and Agora channel ID into appropriate section of [`ClientToken.kt`](./app/src/main/java/com/banuba/sdk/agorapluginexample/ClientToken.kt) with " " symbols.
+14. As a result of the previous two points, in the file [`ClientToken.kt`](./app/src/main/java/com/banuba/sdk/agorapluginexample/ClientToken.kt) you should have the following:
```
Comment (dmitrystoyanovsky) [inline README.md:42]: two previous steps

File: README.md [hunk 4]
```diff
@@ -15,21 +15,41 @@ Examples for [Banuba SDK on Android](https://docs.banuba.com/face-ar-sdk-v1/andr
...
-13. Open the project in Android Studio and run the necessary target using the usual steps.
+   >`banuba_effect_player-release.aar` => [`libs/`](./libs)
+4. Copy `include/bnb` directory into [`libs/bnb_sdk/`](./libs/bnb_sdk):
+   >`include/bnb` => [`libs/bnb_sdk/`](./libs/bnb_sdk)
+5. Download the effects you want from [HERE](https://docs.banuba.com/face-ar-sdk-v1/overview/demo_face_filters). Unzip the downloaded `.zip` files with effects to a folder [`app/src/main/assets/effects`](./app/src/main/assets/effects). Each effect is one folder.
+6. Get the Agora Video SDK archive for android (v4.0.0-beta.2) from [HERE](https://docs.agora.io/en/extension_customer/downloads?platform=Android).
+7. Copy `.jar` files from the Agora Video SDK archive into [`libs`](./libs) dir:
+   >`agora-rtc-sdk.jar` => [`libs/`](./libs)
+8. Copy architecture folders from the Agora Video SDK archive into [`jniLibs`](./libs/jniLibs) dir:
+   >`arm64-v8a`, `armeabi-v7a` and `x86_64` => [`libs/jniLibs/`](./libs/jniLibs)
```
Comment (sdk-banuba) [inline README.md:26]: merge copied folders with existing folders in libs/jniLibs

Comment (mikhailvoronkov1) [inline README.md:26]: merge copied folders with existing folders in libs/jniLibs

**Review by dmitrystoyanovsky — APPROVED** [review]

**Pull request merged**